lib/ucfg.rb

```diff
@@ -1,11 +1,49 @@
 # frozen_string_literal: true
 
 require "ucfg/version"
+require "ostruct"
 
-module Ucfg
+module Ucfg # rubocop:todo Style/Documentation
   class Error < StandardError; end
-  # Your code goes here...
+  # rubocop:todo Metrics/PerceivedComplexity
+  # rubocop:todo Metrics/MethodLength
+  # rubocop:todo Metrics/AbcSize
+  def self.validate(config, schema) # rubocop:todo Metrics/CyclomaticComplexity
+    OpenStruct.new(valid?: true, errors: [])
```

Comment:
We can remove this line as we don't assign the object to a variable, meaning we basically are not using it.

Reply:
Removed line 12